ivy/functional/backends/torch/creation.py

```diff
@@ -373,6 +373,10 @@ def zeros_like(
     return torch.zeros_like(x, device=as_native_dev(device))
 
 
+def from_numpy(x):
```

Comment:
Add input and output type hints here: `from_numpy(x: np.ndarray) -> torch.Tensor`